tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -245,5 +249,25 @@ public bool IsFrameworkAssembly(AssemblyReferenceInformation assembly)
                 return string.Equals(_assemblyName, assembly?.Name);
             }
         }
+
+        private class StreamFromFileAssemblyFile : IAssemblyFile
+        {
+            private readonly byte[] _bytes;
+            private readonly string _path;
+
+            public StreamFromFileAssemblyFile(string path)
+            {
+                _path = path;
+                _bytes = File.ReadAllBytes(path);
+            }
+
+            public bool Exists { get; } = true;
+
+            public string FullName => _path;
```

Comment:
The FullName can be anything - it's required for the analysis to know what the "name" is. We can change it to Path, but conceptually, it's just the name of the input assembly so the reports have something to identify it with. We can call it "Name" since FullName doesn't really mean anything outside of the Path usage